cdap-docs/tools/cdap-default/cdap-default-deprecated.xml

```diff
@@ -139,10 +139,7 @@
     <name>metadata.updates.publish.enabled</name>
     <value>false</value>
     <description>
-      Determines if metadata updates will be published to Apache Kafka.
-      External systems can subscribe to the Kafka topic determined by
-      ${metadata.updates.kafka.topic} to receive notifications of metadata
-      updates (deprecated).
+      Determines if metadata updates will be published to Apache Kafka (deprecated)
```

Comment:
this property was removed from ``cdap-default.xml`` on develop. In fact, all of these ``metadata.updates.*`` properties have been removed now.